shared/chat/conversation/messages/wrapper/shared.js

```diff
@@ -59,15 +59,11 @@ const EditedMark = () => (
 )
 
 const Failure = ({failureDescription, onEdit, onRetry}) => {
-  const error = `${failureDescription ? ` -  ${failureDescription}` : ''}. `
+  const error = `${failureDescription ? `${failureDescription}` : ''}. `
```

Comment:
Looks like this component is only ever created when `failureDescription` is non-empty (see around line 136) so this can be simplified to:
```
const error = `${failureDescription}. `
```